Platform/ja.lproj/Localizable.strings

```diff
@@ -0,0 +1,805 @@
+
+/* No comment provided by engineer. */
+"Icon" = "アイコン";
+
+/* No comment provided by engineer. */
+"Image Type" = "イメージの種類";
+
+/* Import button */
+"Import" = "インポート";
+
+/* No comment provided by engineer. */
+"Import Virtual Machine..." = "VMをインポート..";
+
+/* Save VM overlay */
+"Importing %@..." = "%@をインポート中...";
+
+/* No comment provided by engineer. */
+"Interface" = "インターフェース";
+
+/* UTMQemu */
+"Internal error has occurred." = "内部エラーが発生しました。";
+
+/* UTMVirtualMachine */
+"Internal error starting main loop." = "Internal error starting main loop.";
+
+/* UTMVirtualMachine */
+"Internal error starting VM." = "VMの起動中に内部エラーが発生しました。";
+
+/* VMDisplayMetalWindowController */
+"Internal error." = "内部エラー。";
+
+/* VMConfigSystemViewController */
+"Invalid core count." = "コア数が無効です。";
+
+/* UTMData */
+"Invalid drive size." = "ドライブサイズが無効です。";
+
+/* VMRemovableDrivesViewController */
+"Invalid file selected." = "選択されましたファイルが無効です。";
+
+/* VMConfigSystemViewController */
+"Invalid memory size." = "メモリサイズが無効です。";
+
+/* VMConfigDriveCreateViewController */
+"Invalid name" = "名前が無効です。";
+
+/* VMConfigDriveCreateViewController */
+"Invalid size" = "サイズが無効です。";
+
+/* VMListViewController */
+"Invalid UTM not imported." = "無効なUTMがインポートされていません。";
+
+/* No comment provided by engineer. */
+"Invert Mouse Scroll" = "マウススクロールを反転";
+
+/* No comment provided by engineer. */
+"IP Configuration" = "IPの設定";
+
+/* No comment provided by engineer. */
+"Isolate Guest from Host" = "ゲストをホストから隔離する";
+
+/* No comment provided by engineer. */
+"JIT Cache" = "JITキャッシュ";
+
+/* VMConfigSystemViewController */
+"JIT cache size cannot be larger than 2GB." = "JITキャッシュサイズは2GBを超えることはできません。";
+
+/* VMConfigSystemViewController */
+"JIT cache size too small." = "JITキャッシュサイズが小さすぎます。";
+
+/* No comment provided by engineer. */
+"Legacy" = "レガシ";
+
+/* No comment provided by engineer. */
+"Legacy (PS/2) Mode" = "レガシ (PS/2) モート";
+
+/* No comment provided by engineer. */
+"License" = "ライセンス";
+
+/* UTMConfiguration */
+"Linear" = "Linear";
+
+/* UTMConfiguration */
+"Linux Device Tree Binary" = "Linux Device Tree Binary";
+
+/* UTMConfiguration */
+"Linux Kernel" = "Linux Kernel";
+
+/* UTMConfiguration */
+"Linux RAM Disk" = "Linux RAM Disk";
+
+/* No comment provided by engineer. */
+"Logging" = "Logging";
+
+/* UTMQemuManager */
+"Manager being deallocated, killing pending RPC." = "Manager being deallocated, killing pending RPC.";
+
+/* No comment provided by engineer. */
+"MB" = "MB";
+
+/* No comment provided by engineer. */
+"Memory" = "メモリー";
+
+/* VMDisplayMetalWindowController */
+"Metal is not supported on this device. Cannot render display." = "Metal is not supported on this device. Cannot render display.";
+
+/* No comment provided by engineer. */
+"Mouse Wheel" = "マウスホイール";
+
+/* Save VM overlay */
+"Moving %@..." = "%@移動中...";
+
+/* Clone VM name prompt title */
+"Name" = "名前";
+
+/* VMConfigInfoView */
+"Name is an invalid filename." = "無効なファイル名前";
```

Comment:
- It should be "ファイル名" instead of "ファイル名前". In a compound, a word with kun-yomi ("名前" = "なまえ") is avoided and a word or kanji with on-yomi ("名" = "めい") is preferred.
- It should be written as a sentence as the original text is.

Reply:
ファイル名は無効です